docs/alerting/monitors.md

```diff
@@ -32,15 +32,62 @@ Destination | A reusable location for an action, such as Amazon Chime, Slack, or
 
-   This information is stored in plain text in the Elasticsearch cluster. We will improve this design in the future, but for now, the encoded credentials (which are neither encrypted nor hashed) might be visible to other Elasticsearch users.
+This information is stored in plain text in the Elasticsearch cluster. We will improve this design in the future, but for now, the encoded credentials (which are neither encrypted nor hashed) might be visible to other Elasticsearch users.
 
+###  Email as a destination
+To send or receive an alert notification as an email, first select **Email** as the destination **Type** for the alert. Next, you must add at least one sender and a recipient. We also recommend adding email groups if you want to notify more than a few people of an alert. You can configure senders and recipients using **Manage senders** and **Manage email groups**. 
+
+#### Manage senders
+Manage senders allows you to configure and manage **Sender** email addresses. Senders are email accounts from where the alert notification is sent to different recipients or email groups. 
+
+To configure a sender email, do the following:
+
+1. Choose **Manage senders**.
+1. In the **Manage email senders** modal window, choose **Add sender**, **New sender**. Multiple senders can be added one at a time.
+1. Enter a unique **Sender name**.
+1. Enter the **Email address**, SMTP **Host** (e.g. `smtp.gmail.com` for a Gmail account), and the **Port** number.
+1. You can choose to use an **Encryption method** or leave it as **None**. However, most email providers require SSL or TLS and this requires you to add a username and password to the Elasticsearch keystore. Refer to [Authenticate sender account](#authenticate-sender-account) to learn more.
+1. Choose **Save** to save the configuration and create the sender. You can create a sender even before you enter your credentials for SSL or TLS. However, you must [authentcate each sender account](#authenticate-sender-account) with credentials before you use the destination to send your alert. 
+
+Once the sender is created, the sender account is available to be selected when creating an email destination. You can reuse senders across many different destinations, but each destination only supports one sender. 
+
+#### Manage email groups or recipients 
+Use email groups to create and manage reusable lists of email addresses. For example, one alert might email the DevOps team, whereas another might email the DevOps team and the engineering team.
+
+You can enter individual email addresses, or an email group in the **Recipients** field. For email groups, you can pre-create a group using **Manage email groups**.
+
+To create and manage email groups, do the following:
+1. Choose **Manage email groups**, and then **Add email group**, **New email group**.
```

Comment:
**Actions**, **Manage email groups**

Reply:
I've changed it like this: Once you select **Email** as the destination **Type**, choose **Manage email groups**, and then **Add email group**, **New email group**. You can also do this using the **Actions** button on the top right of the **Destinations** page.